### PR TITLE
new package: lzop

### PR DESCRIPTION
**Pull request by arco**
Needed for those that wants to compress the kernel with LZO

Add the following to your kernel entry in package.mk:

PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET lzop:host"

### CODE REVIEW

File: packages/compress/lzop/package.mk [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+PKG_NAME="lzop"
+PKG_VERSION="1.03"
+PKG_REV="1"
+PKG_ARCH="any"
+PKG_LICENSE="GPL"
+PKG_SITE="http://www.lzop.org/"
+PKG_URL="http://www.lzop.org/download/$PKG_NAME-$PKG_VERSION.tar.gz"
+PKG_DEPENDS_HOST="lzo:host"
+PKG_DEPENDS_TARGET="toolchain"
```
Comment (stefansaraev) [inline packages/compress/lzop/package.mk]: PKG_DEPENDS_HOST="" and remove PKG_DEPENDS_TARGET

**Comment by stefansaraev** [comment]
imo, packages that are referenced anywhere in the buildsystem do not belong in OE tree. I just sent a huge python clean up pr, and I plan more ;)

File: packages/compress/lzop/package.mk (hunk 1 above)
Comment (stefansaraev) [inline packages/compress/lzop/package.mk]: sorry. ignore the PKG_DEPENDS_HOST part. I am blind.

**Comment by arco** [comment]
Yup, that's true. But building the kernel will fail if they have selected LZO as the compression, and they don't have lzop installed in their system. Maybe the package should be a build dependency instead?

**Comment by stefansaraev** [comment]
up to @sraue if he wants this.. 
honestly, I dont care much. for me. gzip is enough

**Comment by sraue** [comment]
how much benefit you have from lzo support? and which system?

**Comment by arco** [comment]
Decompression speed is faster, at the cost of slightly larger file. LZ4 is even better though, but LZO should be supported in all kernels since 2.6.33 I think.

I definitely notices an improvement with both on my Amlogic S805 based stick, but I settled with using LZ4 though, since it seems to perform the best overall. So if you don't want this in, it's ok for me. :)

**Comment by chewitt** [comment]
I'm closing this PR because if LZ4 is all-round better than LZO we wouldn't bother with LZO, thanks.

**Pull request closed without merging**